src/library/simplified/extension/marketingABTests.js

```diff
@@ -0,0 +1,13 @@
+var abTests = utag_data.context.tests;
+var marketingABTests = "";
+
+if (abTests != undefined) {
+    for (var abTest=0; abTest<abTests.length; abTest++) {
+            var experimentId = utag_data.context.tests[abTest].experimentId;
+            var bucketValue = utag_data.context.tests[abTest].bucketValue;
+            marketingABTests += experimentId + "." + bucketValue + "|";
+    }
+}
+
+marketingABTests = marketingABTests.substring(0,marketingABTests.length - 1);
```

Comment:
This is not necessary. one less instruction will help speed things up especially we don't want the page to be slow, but you can leave it as is if you want :).